presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlIntegrationSmokeTest.java

```diff
@@ -180,7 +181,7 @@ public void testInsertIntoNotNullColumn()
         assertUpdate(createTableSql);
         assertEquals(computeScalar("SHOW CREATE TABLE test_insert_not_null"), createTableSql);
 
-        assertQueryFails("INSERT INTO test_insert_not_null (column_a) VALUES (date '2012-12-31')", "Column 'column_b' cannot be null");
+        assertQueryFails("INSERT INTO test_insert_not_null (column_a) VALUES (date '2012-12-31')", "Field 'column_b' doesn't have a default value");
```

Comment:
Where does "Field" come from?
This is probably exception coming directly from mysql. I think we should prefix the message with something of ours, otherwise the err is not searchable in the codebase.